docs/api.md

```diff
@@ -355,7 +355,7 @@ console.log(await page.evaluate('location.href'));
   - `expires` <[number]> Unix time in seconds.
   - `httpOnly` <[boolean]>
   - `secure` <[boolean]>
-  - `sameSite` <"Strict"|"Lax"|"None">
+  - `sameSite` <"Lax"|"None"|"Strict">
```

Comment:
Is there a chance we could leave arbitrary order in the docs and then ignore order when comparing?

Reply:
Done.